x-pack/plugins/maps/public/classes/sources/es_geo_line_source/es_geo_line_source.tsx

```diff
@@ -25,6 +20,8 @@ import { ESDocField } from '../../fields/es_doc_field';
 import { UpdateSourceEditor } from './update_source_editor';
 import { SourceEditorArgs } from '../source';
 
+const MAX_TRACKS = 100;
```

Comment:
@nreese is this figure final? does this mean that Elastic Maps won't ask for more than 100 tracks in a single request right?

Reply:
100 is an arbitrary value. This can be adjusted as we conduct testing to see what value makes sense without overloading elasticsearch. Another solution to get more tracks would be to use composite aggregation and page through tracks. Either way, you will not be able to get 10,000s of tracks since they consume so much memory on elasticsearch. I was getting out of memory errors when requesting 200 tracks at a time.

Reply:
Thanks for the clarification, 100 to 200 tracks feel low to me. We need to see how the number of buckets and documents per bucket play together.